model/project_test.go

```diff
@@ -804,6 +804,31 @@ func (s *projectSuite) TestBuildProjectTVPairs() {
 	s.Len(patchDoc.Tasks, 6)
 }
 
+func (s *projectSuite) TestResolvePatchVTs() {
```

Comment:
I guess it's fine to not test the code unrelated to this change, but can we test other regex cases:
* Using the special "all" specifier ignores regexes and just gets all the tasks/variants.
* Using a mix of names + regexes combines the results.
* Using an alias + regex combines the results.

Reply:
added more tests